apps/web/playwright/teams.e2e.ts

```diff
@@ -135,6 +135,43 @@ test.describe("Teams", () => {
     expect(teamMatesObj.some(({ name }) => name === chosenUser)).toBe(true);
     // TODO: Assert whether the user received an email
   });
+  test("Non admin team members cannot create team in org", async ({ page, users }) => {
+    const teamMateNames = ["teammate-1", "teammate-2", "teammate-3", "teammate-4"];
+    const teamMatesObj = teamMateNames.map((name) => {
+      return { name };
+    });
+    const owner = await users.create(undefined, {
+      hasTeam: true,
+      isOrg: true,
+      teammates: teamMatesObj,
+    });
+
+    const allUsers = await users.get();
+
+    const memberUser = allUsers.find((user) => user.name === teamMateNames[0]);
```

Comment:
hey there, I'm not sure to understand here why we create the teamMateNames array and then only check for teamMateNames[0]? do we even need an array ?

Reply:
we can just have have one teammate  instead of four.